src/app/backend/dashboard.go

```diff
@@ -116,8 +121,15 @@ func main() {
 
 	// Init integrations
 	integrationManager := integration.NewIntegrationManager(clientManager)
-	integrationManager.Metric().ConfigureHeapster(args.Holder.GetHeapsterHost()).
-		EnableWithRetry(integrationapi.HeapsterIntegrationID, time.Duration(args.Holder.GetMetricClientCheckPeriod()))
+
+	if args.Holder.GetMetricsProvider() == "sidecar" {
```

Comment:
This could be a switch since only 1 metric provider can be active anyway

Reply:
Done!